packages/cyberstorm/src/components/componentStyles/Tooltip.module.css

```diff
@@ -0,0 +1,92 @@
+/* reset */
+button {
+  all: unset;
+}
+
+.tooltip {
+  padding: 0.5rem 0.75rem;
+  border-radius: 0.5rem;
+  color: var(--color-white);
+
+  font-weight: 700;
```

Comment:
I think the whole file needs refactoring when the global.css changes drop from the PackageCard PR. Use variables and better naming and stuff.